docs/docs/launch-manual/sections/using-pulsar/sections/editing-and-deleting-text.md

```diff
@@ -0,0 +1,243 @@
+
+You can also delete or cut text out of your buffer with some shortcuts.
+Be ruthless.
+
+::: tabs#using-pulsar
+
+@tab Linux
+
+- <kbd>Ctrl+Shift+K</kbd> - Delete current line
+- <kbd>Ctrl+Backspace</kbd> - Delete to beginning of word
+- <kbd>Ctrl+Delete</kbd> - Delete to end of word
+
+@tab macOS
+
+- <kbd>Ctrl+Shift+K</kbd> - Delete current line
+- <kbd>Alt+Backspace</kbd> or <kbd>Alt+H</kbd> - Delete to beginning of word
+- <kbd>Alt+Delete</kbd> or <kbd>Alt+D</kbd> - Delete to end of word
+- <kbd>Cmd+Delete</kbd> - Delete to end of line
+- <kbd>Ctrl+K</kbd> - Cut to end of line
+- <kbd>Cmd+Backspace</kbd> - Delete to beginning of line
+
+@tab Windows
+
+- <kbd>Ctrl+Shift+K</kbd> - Delete current line
+- <kbd>Ctrl+Backspace</kbd> - Delete to beginning of word
+- <kbd>Ctrl+Delete</kbd> - Delete to end of word
+
+:::
+
+#### Multiple Cursors and Selections
+
+One of the cool things that Pulsar can do out of the box is support multiple
+cursors. This can be incredibly helpful in manipulating long lists of text.
+
+::: tabs#using-pulsar
+
+@tab Linux
+
+- <kbd>Ctrl+Click</kbd> - Add a new cursor at the clicked location
+- <kbd>Alt+Shift+Up/Down</kbd> - Add another cursor above/below the current
+  cursor
+- <kbd>Ctrl+D</kbd> - Select the next word in the document that is the same as
+  the currently selected word
+- <kbd>Alt+F3</kbd> - Select all words in the document that are the same as the
+  currently selected word
+
+@tab macOS
+
+- <kbd>Cmd+Click</kbd> - Add a new cursor at the clicked location
+- <kbd>Ctrl+Shift+Up/Down</kbd> - Add another cursor above/below the current
+  cursor
+- <kbd>Cmd+D</kbd> - Select the next word in the document that is the same as
+  the currently selected word
+- <kbd>Cmd+Ctrl+G</kbd> - Select all words in the document that are the same as
+  the currently selected word
+- <kbd>Cmd+Shift+L</kbd> - Convert a multi-line selection into multiple cursors
+
+@tab Windows
+
+- <kbd>Ctrl+Click</kbd> - Add a new cursor at the clicked location
+- <kbd>Alt+Ctrl+Up/Down</kbd> - Add another cursor above/below the current
+  cursor
+- <kbd>Ctrl+D</kbd> - Select the next word in the document that is the same as
+  the currently selected word
+- <kbd>Alt+F3</kbd> - Select all words in the document that are the same as the
+  currently selected word
+
+:::
+
+Using these commands you can place cursors in multiple places in your document
+and effectively execute the same commands in multiple places at once.
+
+![Using multiple cursors](@images/atom/multiple-cursors.gif)
+
+This can be incredibly helpful in doing many type of repetitive tasks such as
+renaming variables or changing the format of some text. You can use this with
+almost any package or command - for example, changing case and moving or
+duplicating lines.
+
+You can also use the mouse to select text with the
+**_LNX/WIN_**: <kbd>Ctrl</kbd> -
+**_MAC_**: <kbd>Cmd</kbd> key pressed down to select multiple regions of your
+text simultaneously.
+
```

Comment:
```suggestion

Return to a single cursor with <kbd>Esc</kbd> or by clicking anywhere in the file to position a cursor there as normal.

```

https://github.com/atom/flight-manual.atom.io/pull/693/files
(btw, I can't access discord right now so if there's anything you want to tell me comment on github)

Reply:
I think this is a nice clarification, thanks.